src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package com.example.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull(message = " value can't be null")
     private String title;
+    @NotNull(message = " value can't be null")
     private String author;
```

Comment:
Please consider the moment when the client will send an empty string, like title = "";